scRNA-seq-advanced/setup/rms-integration/config.yaml

```diff
@@ -0,0 +1,8 @@
+# The main directory where processed files will be stored
+base_dir: /shared/data/training-modules/scRNA-seq-advanced/data/rms
+
+# Libraries to process
+libraries: ["SCPCL000478", "SCPCL000479", "SCPCL000480", "SCPCL000481"]
```

Comment:
Can we expand this to all libraries in the project to prep for Ally's work? (An advantage of snakemake is that we won't have to reprocess the ones that are already done!)

Reply:
Yes, I'll update this to cover everything in both our modules.
Of course I just left a review over in the DGE setup notebook about how we might structure `setup/` directories, but this might change it! Tagging in @allyhawkins in case.

Reply:
Ok, here's where this has landed for now - I've updated setup directory name to `setup/rms/` so that we can also use this same directory and its Snakefile for DGE setup. Currently it's processing all integration and DGE module libraries and sending processed output to `/shared/data/training-modules/scRNA-seq-advanced/data/rms/processed_libraries/`. 

I'd envision that the future PR for #572 will tack on more setup here, e.g. add on another rule into the Snakefile and/or include a script to further process the DGE libraries (e.g. integrate them) for use during training. But, the script that runs integration there would not be able to run on one SCE at a time, so strategy on next processing steps is a bit TBD!

Either way the first step is to see whether we like the overall concept of changes in 1df6191 as something to build off of.

Reply:
This is what I had envisioned, where all of the libraries would be pre-processed through this same script that you have in the first rule and then we will have a separate integration rule. I think we could figure it out so that it requires all of the files from the first rule to be complete and use that as input to perform integration, but that's for a separate PR.

Reply:
This should be pretty simple to do: we just leave all all of the files for the first rule in the `target` rule and then add the integrated dataset we want as another input for `target`. (but yes, next PR)